lib/kitchen/driver/openstack.rb

```diff
@@ -400,11 +404,34 @@ def disable_ssl_validation
         Excon.defaults[:ssl_verify_peer] = false
       end
 
-      def wait_for_server(state)
-        if config[:server_wait]
```

Comment:
Should we leave this config option in? server_wait?  If so, should we call it after the new console log read code?  

* Use case I could think of:  Imagine userdata on a Windows box joins the domain and schedules a reboot.  Well the script will schedule the reboot, but might need to let the rest of cloudbase-init to execute.  If it schedules the reboot, but the winrm code comes in, that is still a race condition.  So my first thought is we still might need a generic wait after cloudbase-init is done running in this case.  Just to allow userdata to do something.


Thoughts?